x/upgrade/go.mod

```diff
@@ -1,6 +1,8 @@
 module cosmossdk.io/x/upgrade
 
-go 1.23.1
+go 1.23.2
+
+toolchain go1.23.3
```

Comment:
can we kill all toolchain addition?

Reply:
Yeah, is there any way we can avoid go-mod-tidy-all from adding these?

Reply:
Removed it manually, there's no way to avoid adding it with go mod tidy :/